tests/lib/external/snapd-testing-tools/tests/tests.pkgs/task.yaml

```diff
@@ -10,9 +10,9 @@ execute: |
 
     # pkgs tool presents the usage screen when invoked without arguments
     # or with the -h or --help options.
-    tests.pkgs | MATCH 'usage: tests.pkgs {install,remove} \[PACKAGE...\]'
-    tests.pkgs -h | MATCH 'usage: tests.pkgs {install,remove} \[PACKAGE...\]'
-    tests.pkgs --help | MATCH 'usage: tests.pkgs {install,remove} \[PACKAGE...\]'
+    tests.pkgs | MATCH 'usage: tests.pkgs install \[--no-install-recommends\] \[PACKAGE...\]'
+    tests.pkgs -h | MATCH 'usage: tests.pkgs install \[--no-install-recommends\] \[PACKAGE...\]'
+    tests.pkgs --help | MATCH 'usage: tests.pkgs install \[--no-install-recommends\] \[PACKAGE...\]'
 
     # Update the packages db
```

Comment:
Is this comment relevant?

Reply:
no, I'll remove it, thanks for reviewing